chore: rebalance embedded login messaging and add hybrid-use framing (#1327)
auth0/docs-v2: main/docs/authenticate/login/universal-vs-embedded-login.mdx

| **Feature management** | Features are managed centrally in the Auth0 Dashboard or through the Auth0 Management API and apply to all applications using Universal Login.<br/><br/>For example, when you enable MFA in the Dashboard, it is immediately reflected in subsequent logins. | Features are configured per application, giving you control over the release cadence in each app.<br/><br/>Server-side capabilities such as [API-driven MFA](/docs/secure/multi-factor-authentication) can be enabled without client updates. UI-affecting changes (for example, a new login screen) are released on your application's own schedule. |
| **User experience** | Users are redirected between your application and the Auth0 Authorization Server during authentication.<br/><br/>For native applications, this requires implementation of universal/deep links. | Users remain inside your application during authentication, which can be useful for tightly branded or in-app sign-in experiences. |

> | **Security and maintenance** | Auth0 delivers security updates to the login experience transparently. | Auth0 provides the security building blocks—custom domains, [Cross-Origin Authentication](/docs/authenticate/login/cross-origin-authentication), API-driven MFA, [Native Passkey APIs](/docs/authenticate/database-connections/passkeys) and [DPoP](/docs/secure/sender-constraining/demonstrating-proof-of-possession-dpop)—and you choose when to roll updates out to your users. |

Reviewer: ```suggestion
| [Single Sign-on](/docs/authenticate/single-sign-on) | Built-in cross-application SSO through [session cookies](/docs/manage-users/sessions) on the Auth0 Authorization Server (your Auth0 tenant), including support for third-party applications and multi-domain architectures. | Supported with shared domain architecture. Web applications using the [Lock](/docs/libraries/lock/lock-api-reference#checksession) or [Auth0.js](/docs/libraries/auth0js#using-checksession-to-acquire-new-tokens) libraries can share sessions when configured under the same custom domain. Native applications can share sessions with web applications through [Native to Web SSO](/docs/authenticate/single-sign-on/native-to-web). |
| [Customization](/docs/customize) | Customize many parts of the experience through theming, page templates, text elements, and prompts, or use [Advanced Customizations for Universal Login](/docs/customize/login-pages/advanced-customizations) for full UI customization. | Full control over the UI because the login form is rendered by your application. |
| Feature management | Features are managed centrally in the Auth0 Dashboard or through the Auth0 Management API and apply to all applications using Universal Login.<br/><br/>For example, when you enable MFA in the Dashboard, it is immediately reflected in subsequent logins. | Features are configured per application, giving you control over the release cadence in each app.<br/><br/>Server-side capabilities such as [API-driven MFA](/docs/secure/multi-factor-authentication) can be enabled without client updates. UI-affecting changes (for example, a new login screen) are released on your application's own schedule. |
| User experience*| Users are redirected between your application and the Auth0 Authorization Server during authentication.<br/><br/>For native applications, this requires implementation of universal/deep links. | Users remain inside your application during authentication, which can be useful for tightly branded or in-app sign-in experiences. |
| Security and maintenance | Auth0 delivers security updates to the login experience transparently. | Auth0 provides the security building blocks (like custom domains, [Cross-Origin Authentication](/docs/authenticate/login/cross-origin-authentication), API-driven MFA, [Native Passkey APIs](/docs/authenticate/database-connections/passkeys) and [DPoP](/docs/secure/sender-constraining/demonstrating-proof-of-possession-dpop)) and you choose when to roll updates out to your users. |
```

## Choosing between hosted and embedded login

Hosted and embedded login are not mutually exclusive. The same application can use Universal Login for some flows and embedded flows for others, picking the approach that best fits each part of the user journey.

> * **Universal Login** is a strong default for primary sign-in, sign-up, and password reset, especially when you want Auth0 to manage the login surface, deliver security updates automatically, or coordinate SSO across third-party or multi-domain applications.

Reviewer 2: ```suggestion
* **Universal Login**: Auth0 manages the login surface and is the default for primary sign-in, sign-up, and password reset. Auth0's hosted solution also provides automatic security updates and coordinates SSO across third-party or multi-domain applications.
```

> * **Embedded Login** fits flows where you want full control over the UI or want to keep users inside your application—for example, [factor enrollment](/docs/authenticate/login/embedded-login/use-cases/enrollment), step-up authentication, passkey registration with [Passkey APIs](/docs/authenticate/database-connections/passkeys), [connected account linking](/docs/manage-users/user-accounts/user-account-linking), or sign-in using [Native Social Login](/docs/authenticate/identity-providers/social-identity-providers).

Reviewer 2: ```suggestion
* **Embedded Login**: Allows full control over the UI and keeps users inside your application. You can use Embedded Login for [factor enrollment](/docs/authenticate/login/embedded-login/use-cases/enrollment), step-up authentication, passkey registration with [Passkey APIs](/docs/authenticate/database-connections/passkeys), [link connected accounts](/docs/manage-users/user-accounts/user-account-linking), or sign-in using [Native Social Login](/docs/authenticate/identity-providers/social-identity-providers).
```
